app/controllers/coronavirus_pages_controller.rb

```diff
@@ -42,13 +40,22 @@ def initialise_coronavirus_pages
   end
 
   def coronavirus_page
-    @coronavirus_page ||= updater.page
+    @coronavirus_page ||= model_builder.page
```

Comment:
Fair enough, I guess `page` is the only public method on `model_builder`. Could be worth combining though :-)

Reply:
Yes - good idea